matrix_client/api.py

```diff
@@ -787,3 +787,50 @@ def set_guest_access(self, room_id, guest_access):
             "guest_access": guest_access
         }
         return self.send_state_event(room_id, "m.room.guest_access", content)
+
+    def get_devices(self):
+        """Gets information about all devices for the current user."""
+        return self._send("GET", "/devices")
+
+    def get_device(self, device_id):
+        """Gets information on a single device, by device id."""
+        return self._send("GET", "/devices/%s" % device_id)
+
+    def update_device_info(self, device_id, display_name):
+        """Update the display name of a device.
+
+        Args:
+            device_id (str): The device ID of the device to update.
+            display_name (str): New display name for the device.
+        """
+        content = {
+            "display_name": display_name
+        }
+        return self._send("PUT", "/devices/%s" % device_id, content=content)
+
+    def delete_device(self, auth_body, device_id):
+        """Deletes the given device, and invalidates any access token associated with it.
+        Requires User-Interactive Authentication API.
```

Comment:
Sorry I missed this the first time through. Please try to follow [PEP 257 conventions for multi-line docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings). In this case, that means a single summary line separated by a blank new line from a more detailed description. Something like:

```python
"""Deletes the given device and invalidates any associated access tokens.

NOTE: This endpoint uses the User-Interactive Authentication API.

Args:
    auth_body (dict): Authentication params.
    device_id (str): The device ID of the device to delete.
"""
```

Reply:
Thanks, fixed. I mostly know PEP8 through flake8 and tend to miss those kind of things.